golem_verificator/verifier.py

```diff
@@ -52,9 +52,9 @@ def task_timeout(self, subtask_id):
             self.time_started = self.time_ended = datetime.utcnow()
             self.state = SubtaskVerificationState.TIMEOUT
             self.message = "Verification never ran, task timed out"
-        return subtask_id, self.state, self._get_answer()
-
+        
         self._clear_state()
+        return subtask_id, self.state, self._get_answer()
```

Comment:
You should make a copy of state and answer, before calling `_clear_state`, because that's returning cleared state.